docs/guides/deployment/storage-guide.md

```diff
@@ -198,7 +198,7 @@ Another popular way to store your flow code is to include it in a Docker image.
     - Azure Container Instances - Push
     - Google Cloud Run - Push
 
-1. Run `prefect init` in the root of your repository and choose `docker` for the project name and answer the prompts to create a `prefect.yaml` file with a build step that will create a Docker image with the flow code built in. See the [Docker guide](/guides/deployment/docker/) for more info.
+1. Run `prefect init` in the root of your repository and choose `docker` for the project name and answer the prompts to create a `prefect.yaml` file with a build step that will create a Docker image with the flow code built in. See the [Docker guide](/tutorial/worker/) for more info.
```

Comment:
```suggestion
1. Run `prefect init` in the root of your repository and choose `docker` for the project name and answer the prompts to create a `prefect.yaml` file with a build step that will create a Docker image with the flow code built in. See the [Workers and Work Pools page of the tutorial](/tutorial/worker/) for more info.
```

Reply:
Looks like it's a link to the tutorial page instead of the guide, so updated the text.

Reply:
Fixed a typo while you were writing this but it's in the next commit.